packages/eslint-plugin-pf-codemods/src/rules/v6/mastheadNameChanges/mastheadNameChangesOutput.tsx

```diff
@@ -0,0 +1,7 @@
+import { MastheadLogo, MastheadBrand } from "@patternfly/react-core";
```

Comment:
This output just needs to be re-updated, looks like it's still using the fixed output from when you were also updating MastheadMain to MastheadBrand

Reply:
Whoops